examples/react-urql-retry/README.md

```diff
@@ -0,0 +1,15 @@
+# Integrating with React
+
+Integrating urql is as simple as:
+
+1.  Install packages
+
+yarn add urql @urql/exchange-retry graphql
```

Comment:
Let's wrap this one in backticks as well, makes it easier for folks to copy-paste